tests/integration/components/settings/application-page-test.js

```diff
@@ -0,0 +1,10 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('settings/application-section', 'Integration | Component | settings/application page');
```

Comment:
Change `Integration | Component | settings/application page` to `Integration | Component | settings/application section` to match the component name